### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by avahoffman**
Pretty self explanatory :)

### CODE REVIEW

**Comment by avahoffman** [comment]
@KatherineCox fyi! Please feel free to change anything

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
 <!--Delete this section below upon using the template-->
 
-## Online Tools for Training Resources - OTTR
+## AnVIL Template
 
-The purpose of this course template and guide is to make course publishing maintenance across multiple different platforms _less painful_.
+Based on _Online Tools for Training Resources (OTTR)_
```
Comment (KatherineCox) [inline README.md:5]:
```suggestion
Based on [_Online Tools for Training Resources (OTTR)_](https://github.com/jhudsl/OTTR_Template)
```

File: README.md [hunk 2]
```diff
@@ -16,29 +18,21 @@ _This template and guide helps you_:
...
-- [Reproducibility in Cancer Informatics](https://jhudatascience.org/Reproducibility_in_Cancer_Informatics/)
-- [Advanced Reproducibility in Cancer Informatics](https://jhudatascience.org/Adv_Reproducibility_in_Cancer_Informatics/)
-
 ## To Get Started
-- Go to the [Wiki page](https://github.com/jhudsl/OTTR_Template/wiki/Getting-started)
+- Go to the [OTTR Wiki page](https://github.com/jhudsl/OTTR_Template/wiki/Getting-started)
 - Please take a look at the [code of conduct](./code_of_conduct.md).
-- If you encounter any problems or have ideas for improvements to this template repository or this getting started guide, please [file an issue here](https://github.com/jhudsl/OTTR_Template/issues/new/choose)! Your feedback is very much appreciated.
+- If you encounter any problems or have improvement ideas for this template repository, please [file an issue](https://github.com/jhudsl/AnVIL_bookdown_style/issues/new/choose)! Issues for the parent repository (the OTTR template repository) is also very much appreciated and can be submitted [here](https://github.com/jhudsl/OTTR_Template/issues/new/choose).
```
Comment (KatherineCox) [inline README.md:24]:
```suggestion
- If you encounter any problems or have improvement ideas for this template repository, please [file an issue](https://github.com/jhudsl/AnVIL_bookdown_style/issues/new/choose)! Issues for the parent repository (the OTTR template repository) are also very much appreciated and can be submitted [here](https://github.com/jhudsl/OTTR_Template/issues/new/choose).
```

File: README.md [hunk 3]
```diff
@@ -16,29 +18,21 @@ _This template and guide helps you_:
...
 
 <!--Delete everything above this line upon using the template-->
 
 # {Course Name}
 
 [![Render Bookdown and Coursera](https://github.com/jhudsl/OTTR_Template/actions/workflows/render-all.yml/badge.svg)](https://github.com/jhudsl/OTTR_Template/actions/workflows/render-all.yml)
 
-This course was created from [this GitHub template](https://github.com/jhudsl/OTTR_Template).
+This course was created from [the AnVIL Template](https://github.com/jhudsl/AnVIL_bookdown_style).
```
Comment (KatherineCox) [inline README.md:32]:
```suggestion
This course was created from [the AnVIL Template](https://github.com/jhudsl/AnVIL_bookdown_style), which is based on the [OTTR GitHub template](https://github.com/jhudsl/OTTR_Template)
```

**Review by KatherineCox — APPROVED** [review]
Caught a typo and suggested adding a couple of link-backs to OTTR.  Thanks for filling this in!

**Pull request merged**